README.md

```diff
@@ -1,2 +1,89 @@
 # nomad-plugin-template
 A template repository for creating a repository with a NOMAD plugin package.
+
+> [!WARNING]
+> The template is not yet ready for production usage.
+
+
+## Getting started
+
+1. Click on the `Use this template` button and create a new plugin repository. The form will ask you to fill out the name for the new plugin repository.
+
+2. In the newly created repository, start a new Github Codespace and generate the plugin structure.
+
+Run the following command to create a new Nomad plugin project using cookiecutter-nomad-plugin:
+
+```sh
+cruft create https://github.com/FAIRmat-NFDI/cookiecutter-nomad-plugin
+```
+
+Cookiecutter prompts you for information regarding your plugin:
+
+```no-highlight
+full_name [John Doe]: Citizen Kane
+email [john.doe@physik.hu-berlin.de]: citizen@kane.de
+github_username [foo]: kane
+plugin_name [foobar]: awesome
+module_name [awesome]: awesome
```

Comment:
What efffect do `module_name` and 'plugin_name' have? I would have expected that you only put in here the python package name, since we anyways ignore the top level folder.

Reply:
I've changed the plugin_name response from `awesome` to `awesome-tools` to make this distinction clearer. The `module_name` gets populated automatically, so the user just has to press enter there.

> we anyways ignore the top level folder.

Might not be the case if a user runs the cookiecutter template locally, in those cases the user would create a new git repository in the top level folder.